src/core/classes/gas-cost-estimator.ts

```diff
@@ -39,6 +39,31 @@ export class GasCostEstimator<TContract extends BaseContract> {
     return ethers.utils.formatEther(gasUnits.mul(price));
   }
 
+  /**
+   * Estimates the gas limit of a transaction
+   * Pass in the same parameters as the contract's function.
+   * @remarks Estimates the gas limit of a transaction
+   * @example
+   * ```javascript
+   * const gasLimitOfClaim = await nftDrop?.estimator.gasLimitOf("claim", [
+   *   "0x...", // receiver
+   *   1, // quantity
+   *   "0x...", // currency
+   *   1, // price per token
+   *   [], // proofs
+   *   1, // proof max quantity per transaction
+   * ]);
+   * ```
+   * @returns the estimated gas limit in GWEI
```

Comment:
comment is wrong

Reply:
What about it were you referring to - something besides `wei` instead of `gwei`?

Reply:
ah yes - just gas units